Documentation/Columns/_Properties/_Config.rst.txt

```diff
@@ -17,18 +17,18 @@
 
     ..  confval:: type
         :name: columns-config-type
+        :Type: string, one of the :ref:`column types <columns-types>`
```

Comment:
Why upper-case `Type`?

Docs says they all are lower-case: https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Reference/ReStructuredText/Code/Confval.html#confval-directive-api

Reply:
Required and Path are also uppercased?

Reply:
By default required, default and type are key words that should be written in lower case, however we have some manuals that we didnt convert to the standard yet. We can make a mass commit some time, and leave it as-is in this PR. 

All other key are passed on as they are written